ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/execution/InsertValuesExecutorTest.java

```diff
@@ -233,6 +233,27 @@ public void shouldFillInRowtime() {
     verify(producer).send(new ProducerRecord<>(TOPIC_NAME, null, 1L, KEY, VALUE));
   }
 
+  @Test
+  public void shouldHandleRowTimeWithoutRowKey() {
```

Comment:
did you mean "withInferredRowKey"? To have "noRowKey" you need to omit `COL0`

Reply:
I mean the list of fields contains `ROWTIME` but not `ROWKEY`, hence `handleRowTimeWithoutRowKey`...